src/components/LocationFormFragment.tsx

```diff
@@ -0,0 +1,56 @@
+import { Form, Input } from 'antd';
+import React from 'react';
+
+// TODO: use this fragment in signup form, note that "zip" was renamed to "zipCode"
```

Comment:
Leaving TODOs in code? I think in this case its ok because refactoring signup form should be its own PR